modules/sdk-coin-apt/src/lib/transaction/transaction.ts

```diff
@@ -335,8 +349,10 @@ export abstract class Transaction extends BaseTransaction {
       'type',
     ];
 
-    const outputs: TransactionRecipient[] = [this.recipient];
-    const outputAmount = outputs[0].amount;
+    const outputs: TransactionRecipient[] = this._recipients;
```

Comment:
We can change `_recipients` to `recipients` in every place which will use getter instead of accessing the field directly

```suggestion
    const outputs: TransactionRecipient[] = this.recipients;
```

Reply:
Accessing a protected field `_recipients` in the same class. I don't think its a issue.
All the fields like `_sender`,` _maxGasAmount`, `_expirationTime` , etc. all are directly accessed, which is never a issue in the same class.

Reply:
I agree with @at31416